pandas/tests/computation/test_eval.py

```diff
@@ -298,10 +298,9 @@ def check_alignment(self, result, nlhs, ghs, op):
             # TypeError, AttributeError: series or frame with scalar align
             pass
         else:
-
             # direct numpy comparison
             expected = self.ne.evaluate(f"nlhs {op} ghs")
-            tm.assert_numpy_array_equal(result.values, expected)
+            tm.assert_almost_equal(result.values, expected)
```

Comment:
can you add a comment why we are doing this and reference to this issue

Reply:
yah i think this makes sense, but should have a TODO indicating that it should be equality and we had to punt